demos/fleet_provisioning/fleet_provisioning_with_csr/claim_policy.json

```diff
@@ -0,0 +1,31 @@
+ {
+   "Version": "2012-10-17",
+   "Statement": [
+     {
+       "Effect": "Allow",
+       "Action": [
+         "iot:Connect"
+       ],
+       "Resource": "*"
+     },
+     {
+       "Effect": "Allow",
+       "Action": [
+         "iot:Publish",
+         "iot:Receive"
```

Comment:
Why is a separate section for `iot:Subscribe` used when it is using the same resource filters>

Reply:
One uses :topic and the other uses :topicfilter. This is the example template provided by AWS docs with create swapped with create-with-csr

Reply:
Okay cool, no worries then